language/cc/config.go

```diff
@@ -140,6 +147,45 @@ func (c *ccLanguage) Configure(config *config.Config, rel string, f *rule.File)
 			}
 		case cc_unresolved_deps:
 			selectDirectiveChoice(&conf.unresolvedDepsMode, unresolvedDepsModes, d)
+
+		case cc_platform:
+			// Reset existing platforms
+			if d.Value == "" {
+				conf.platforms = map[platform.Platform]platformConfig{}
+				continue
+			}
+			args := strings.Fields(d.Value)
+			if len(args) < 3 {
+				log.Printf("gazelle_cc: invalid %v input: '%v', requires <os> <arch> <constraint> values", d.Key, d.Value)
```

Comment:
Nit: rephrase message to match the README, which seems clearer.

```suggestion
				log.Printf("gazelle_cc: invalid %v input: '%v', requires <os> <arch> <constraint> [macro=value...]", d.Key, d.Value)
```